.gitignore

```diff
@@ -0,0 +1,32 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
This one is quite common. Created if you make a new Github project and choose `Node` as default .gitignore.